clrtrust

```diff
@@ -362,6 +391,11 @@ $tmp"
         return $EERR
     fi
 
+    trap sig_ignore INT HUP TERM
+    if is_system_store && ! lock; then
+        trap - INT HUP TERM
```

Comment:
Add an error message, e.g. 

      1>&2 echo "Error failed to lock $CLR_TRUST_LOCK_FILE"

so the user knows what is happening?

Reply:
Added on the next line.